fighthealthinsurance/utils.py

```
@@ -1180,56 +1213,57 @@ async def best_within_timelimit(
+                    best_score = score
+                    best_result_option = result
+            except Exception as e:
+                _log_fanout_task_error(e, "best_within_timelimit")
+
+    # Main window: wait for everything (or the timeout), take the best.
+    done, pending = await asyncio.wait(
+        wrapped_tasks, timeout=timeout, return_when=asyncio.ALL_COMPLETED
+    )
+    _score_done(done)
     if best_result_option:
         asyncio.create_task(cancel_tasks(list(pending)))
```

Ruff (0.16.0)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)